src/Shared/Components/CICDHistory/History.components.tsx

```diff
@@ -29,47 +36,63 @@ import { ReactComponent as ZoomOut } from '../../../Assets/Icon/ic-exit-fullscre
+        setFullScreenView,
+        shortcut,
+    }: LogResizeButtonType & IWithShortcut): JSX.Element => {
         const { pathname } = useLocation()
-        const zoomButtonRef = useRef<HTMLDivElement>(null)
+        const zoomButtonRef = useRef<HTMLButtonElement>(null)
 
-        const toggleFullScreen = (): void => {
+        const toggleFullScreen = () => {
             // NOTE: need to use ref due to the problem of stale function reference after registering the callback
             setFullScreenView(!(zoomButtonRef.current.dataset.isFullscreenView === 'true'))
         }
 
+        const showButton = pathname.includes('/logs') || !onlyOnLogs
+        const doesShortcutContainCmdKey = shortcutCombo.some((key) => key === 'Control') && IS_PLATFORM_MAC_OS
+
         useEffect(() => {
-            shortcut.registerShortcut(toggleFullScreen, ['f'], 'ToggleFullscreen', 'Enter/Exit fullscreen')
-            shortcut.registerShortcut(
-                () => setFullScreenView(false),
-                ['Escape'],
-                'ToggleFullscreen',
-                'Enter/Exit fullscreen',
-            )
+            const combo = shortcutCombo
+                .map((key) => {
+                    if (key === 'Control') {
+                        return IS_PLATFORM_MAC_OS ? 'cmd' : 'ctrl'
+                    }
+                    return key.toLowerCase()
+                })
+                .join('+')
+
+            // FIXME: disabling shortcut for macos since pressing cmd breaks shortcuts through react-keybind
+            if (showButton && shortcutCombo.length && !doesShortcutContainCmdKey) {
+                shortcut.registerShortcut(toggleFullScreen, [combo], 'ToggleFullscreen', 'Enter/Exit fullscreen')
+            }
 
             return () => {
-                shortcut.unregisterShortcut(['f'])
-                shortcut.unregisterShortcut(['Escape'])
+                shortcut.unregisterShortcut([combo])
             }
-        }, [pathname.includes('/logs')])
+        }, [showButton])
 
         return (
-            pathname.includes('/logs') && (
+            showButton && (
                 <Tooltip
                     placement="left"
                     shortcutKeyCombo={{
                         text: fullScreenView ? 'Exit fullscreen' : 'Enter fullscreen',
-                        combo: ['F'] as const,
+                        combo: doesShortcutContainCmdKey ? null : shortcutCombo,
                     }}
                 >
-                    <div
+                    <button
+                        type="button"
+                        aria-label="Enter/Exit fullscreen"
                         ref={zoomButtonRef}
                         data-is-fullscreen-view={fullScreenView}
-                        className={`zoom ${fullScreenView ? 'zoom--out' : 'zoom--in'} pointer dc__zi-4 flex`}
+                        className={`zoom ${fullScreenView ? 'zoom--out' : 'zoom--in'} pointer dc__zi-4 flex dc__transparent`}
```

Comment:
Please verify once by deploying

Reply:
verified. dc__transparent was not working as you have rightly pointed out. So replaced it with dc__no-border.